api/internal/localizer/locloader_test.go

```diff
@@ -127,65 +127,6 @@ func TestNewMultiple(t *testing.T) {
 	checkLoader(req, higherLdr, "/alpha/beta/say")
 }
 
-func makeWdFs(t *testing.T) map[string]filesys.FileSystem {
-	t.Helper()
-	req := require.New(t)
-
-	root := filesys.MakeEmptyDirInMemory()
-	req.NoError(root.MkdirAll("a/b/c/d/e"))
-
-	outer, err := root.Find("a")
-	req.NoError(err)
-	middle, err := root.Find("a/b/c")
-	req.NoError(err)
-
-	return map[string]filesys.FileSystem{
-		"a":     outer,
-		"a/b/c": middle,
-	}
-}
-
-func TestNewLocLoaderCwdNotRoot(t *testing.T) {
-	cases := map[string]struct {
-		wd     string
-		target string
-		scope  string
-		newDir string
-	}{
-		// target not immediate child of scope
-		"outer dir": {
-			"a",
-			"b/c/d/e",
-			"b/c",
-			"b/newDir",
-		},
-		"scope": {
-			"a/b/c",
-			"d/e",
-			".",
-			"d/e/newDir",
-		},
-	}
-
-	for name, test := range cases {
-		test := test
-		t.Run(name, func(t *testing.T) {
-			req := require.New(t)
-			fSys := makeWdFs(t)[test.wd]
-
-			ldr, args, err := NewLoader(test.target, test.scope, test.newDir, fSys)
-			req.NoError(err)
-			checkLoader(req, ldr, "a/b/c/d/e")
-
-			req.Equal("a/b/c/d/e", args.Target.String())
-			req.Equal("a/b/c", args.Scope.String())
-			req.Equal(test.wd+"/"+test.newDir, args.NewDir.String())
-			// memory file system can only find paths rooted at current node
-			req.True(fSys.Exists(test.newDir))
-		})
-	}
-}
-
```

Comment:
I deleted these tests that use non-root`fsnode`s as the working directory because `fsnode`s only understand relative paths. Before we called `ConfirmDir()` on the established root and repo directories, which are absolute paths, `ConfirmDir()` was only called on relative paths in `fileLoader` in these test cases.

I will cover working directory behavior during integration testing in a future PR.

Reply:
No longer applicable.